CourseChamp/src/test/java/ca/mcgill/ecse428/CourseChamp/Service/ReviewServiceTests.java

```diff
@@ -0,0 +1,64 @@
+package ca.mcgill.ecse428.CourseChamp.Service;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import ca.mcgill.ecse428.CourseChamp.exception.CourseChampException;
+import ca.mcgill.ecse428.CourseChamp.model.Review;
+import ca.mcgill.ecse428.CourseChamp.repository.ReviewRepository;
+import ca.mcgill.ecse428.CourseChamp.service.ReviewService;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpStatus;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReviewServiceTests {
+
+    @Mock
+    private ReviewRepository reviewRepository;
+
+    @InjectMocks
+    private ReviewService reviewService;
+
+    @BeforeEach
+    public void setup() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
You should probably create an instance here or smth that you can use in your tests